libraries/vdf/include/vdf/sortition.hpp

```diff
@@ -62,11 +62,14 @@ class VdfSortition : public vrf_wrapper::VrfSortitionBase {
       "9aee2a207e5173a7ee8f90ee9ab9b6a745d27c6e850e7ca7332388dfef7e5bbe6267d1f7"
       "9f9330e44715b3f2066f903081836c1c83ca29126f8fdc5f5922bf3f9ddb4540171691ac"
       "cc1ef6a34b2a804a18159c89c39b16edee2ede35");
-  bool verifyVrf(const vrf_pk_t& pk, const bytes& vrf_input) const;
+  bool verifyVrf(const vrf_pk_t& pk, const bytes& vrf_input, uint16_t stake) const;
```

Comment:
stake isn't really stake here, but some value proportional to votes/total_votes. I was a little bit confused with it first time because thought that we are passing real stake and should be iterating too much because of it

Reply:
It is stake in promil (part per thousand). I did it this way because I wanted it to not depend on total number of votes and act similar if total votes count is small or large.

Reply:
But you are passing `vote_count * kStakeDivider / total_vote_count` here. So it is not related to stake, but to votes count. Or am I missing something?